.circleci/config.yml

```diff
@@ -1,15 +1,13 @@
 version: 2
 jobs:
-  core:
+  minimal_core:
     machine:
       enabled: true
     steps:
     - checkout
     - run:
-        command: ./gradlew testcontainers:build -x test -x check
+        command: ./gradlew --no-daemon --continue --scan testcontainers:test --tests '*GenericContainerRuleTest'
```

Comment:
We'll want to refine this subset, but for now I think it's a reasonable starting point.